weights/MuPogSF_UL/JSONtoROOT.py

```diff
@@ -0,0 +1,109 @@
+import json
+import numpy as np
+import ROOT as rt
+
+class TrgSF:
+    def __init__(self,name):
+        self.file_name=name
+        self.hist_name={}
+        self.data={}
+        self.nInputs={}
+        self.nScheme=0
+        self.__ReadJSON()
+        
+    def __ReadJSON(self):
+        f=open(self.file_name)
```

Comment:
```f``` is not closed; you can either ```f.close()``` or use a ```with``` statement (better). I think in this case it does make a difference (the function ends immediately and the garbage collector takes care of it), but is a general good practice.